stable/rabbitmq-ha/templates/service.yaml

```diff
@@ -22,7 +22,7 @@ metadata:
 {{ toYaml .Values.extraLabels | indent 4 }}
 {{- end }}
 spec:
-{{- if ne .Values.service.type "NodePort" }}
+{{- if and (ne .Values.service.type "NodePort") (and (ne .Values.service.type "LoadBalancer") (eq .Values.service.clusterIP  "None")) }}
```

Comment:
I don't think this logic is correct, especially the `.Values.service.clusterIP  "None"`. It could be simplified by inverting logic:

`{{- if (or (eq .Values.service.type "ClusterIP") (empty .Values.service.type)) }}`

Reply:
Patch updated.
Thanks